deploy/0.0.1/ci-deploy.sh

```diff
@@ -0,0 +1 @@
+tb deploy --yes
```

Comment:
🚨🚨🚨

We are not populating in CI nor testing anything, please no PR without tests.